### PR TITLE
Add GTC 2020 examples to the GOL-function container

### PR DESCRIPTION
**Pull request by SimeonEhrig**

### CODE REVIEW

**Comment by SimeonEhrig** [comment]
@ax3l 
Can you please overwork the `README.md`

**Comment by ax3l** [comment]
Absolutely, will take a look after getting new groceries.

File: GOL-function-presentation/notebook/GTC_presentations/simulation/color_maps.hpp [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+                                            {237, 77, 62},   //6
+                                            {60, 180, 75},   //7
+                                            {145, 30, 255}}; //8
+    int r(varTyp value){return static_cast<int>(turbo_srgb_bytes[value][0]) * 256;}
+    int g(varTyp value){return static_cast<int>(turbo_srgb_bytes[value][1]) * 256;}
+    int b(varTyp value){return static_cast<int>(turbo_srgb_bytes[value][2]) * 256;}
+};
+ColorHeatMap<int> ch_map;
+
+#endif
```
Comment (ax3l) [inline GOL-function-presentation/notebook/GTC_presentations/simulation/color_maps.hpp]: empty newline missing

File: GOL-function-presentation/notebook/GTC_presentations/simulation/helper.hpp [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+    for(auto c : image){
+        buffer << c;
+    }
+    
+    mine["image/png"] = xtl::base64encode(buffer.str());
+    xeus::get_interpreter().display_data(
+        std::move(mine),
+        xeus::xjson::object(),
+        xeus::xjson::object());
+}
```
Comment (ax3l) [inline GOL-function-presentation/notebook/GTC_presentations/simulation/helper.hpp]: empty newline missing :)

File: GOL-function-presentation/notebook/README.md [hunk 3]
```diff
@@ -0,0 +1,9 @@
+# Content
+
+The notebook `GameOfLife.ipynb` contains a detailed and self-explanatory example of the CUDA extension of Cling. This is the first notebook you should open. The folder `GTC_presentations` contains the example notebooks of the GTC 2020 session `CUDA C++ in Jupyter: Adding CUDA Runtime Support to Cling` [1].
```
Comment (ax3l) [inline GOL-function-presentation/notebook/README.md]:
```suggestion
The notebook `GameOfLife.ipynb` contains a detailed and self-explanatory example of the CUDA extension of Cling. This is the first notebook you should open. The folder `GTC_presentations` contains the example notebooks of the GTC 2020 session `CUDA C++ in Jupyter: Adding CUDA Runtime Support to Cling` [1,2].
```

File: GOL-function-presentation/notebook/README.md [hunk 4]
```diff
@@ -0,0 +1,9 @@
...
+
+[1] https://www.nvidia.com/en-us/gtc/session-catalog/?search=S21588
```
Comment (ax3l) [inline GOL-function-presentation/notebook/README.md:5]:
```suggestion
[1] https://www.nvidia.com/en-us/gtc/session-catalog/?search=S21588
[2] https://doi.org/10.5281/zenodo.3713753
```

**Review by ax3l — APPROVED** [review]
Looks all fantastic, thank you!
I added three small comments inline.

**Pull request merged**